sentry_sdk/tracing_utils.py

```diff
@@ -896,6 +830,88 @@ def _sample_rand_range(parent_sampled, sample_rate):
         return sample_rate, 1.0
 
 
+def create_span_decorator(op=None, name=None, attributes=None):
```

Comment:
Could you please move this function above `get_current_span`? I think it might simplify the diff here